### DISCUSSION TITLE
Function invoke keep failing with cors error 546

### DISCUSSION THREAD

**Discussion by adamcohenhillel** [supabase/supabase#17311] [Questions]
All of a sudden, invoking functions in a specific project stopped working (worked perfectly before). I tried revert changes to different known versions that worked, I tried deleting all functions and deploy them again, I tried to use a known-to-work client-side version to make sure it is not 

The error on the client side is:
```
Access to fetch at 'https://XXX.supabase.co/functions/v1/YYY' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```
And on the edge function side:
<img width="579" alt="image" src="https://github.com/supabase/supabase/assets/633840/3c67c50c-233b-4aa7-9bc0-3441366d176c">
and:
<img width="1236" alt="Screenshot 2023-09-10 at 23 40 09" src="https://github.com/supabase/supabase/assets/633840/e2567700-32df-46e5-a695-b16ed4ef85dd">

I tried to deploy the functions without any changes, and it just failing no matter what - a version that 100% worked, and actually working on a different project, does not work anymore with that specific project.

**Comment by laktek** [comment]
This looks like an issue specific to your function deployment. Closing the discussion in favour of support ticket.

**Reply by adamcohenhillel** [reply]
It's probably an underlying issue with the deployment mechanism, but sure, close the ticket.

**Reply by laktek** [reply]
I've replied to you on the support ticket.
